README.md

```diff
@@ -1,2 +1,174 @@
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add Task**
+- **Remove Task**
+- **Remove Completed Tasks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live demo here](https://Kanyorok.github.io/Capstone-Grp_Kanban/dist)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- A code editor of your choice (like vs code or atom and so on)
+-version control system (git is preferred)
+
+### Setup
+
+- Create a new folder to store the files in.
+
+### Install
+
+Install this project with:
+
+- Open your command prompt or terminal and run.
+- git clone https://github.com/Kanyorok/Capstone-Grp_Kanban.git
+- cd Capstone-Grp_Kanban
+- npm i
+- npm start (To Launch App).
+
+
+### Usage
+
+- this is a Website Application app and I will use it to insert my upcoming projects.
+
+### Run tests
+
+To run tests, run the following command:
+
+- npm test
+- npm lint
+- npm start
+
+### Deployment
+
+You can deploy this project using:
+
+```
+  github pages
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1: Robert Kanyoro**
+
+- GitHub: [@RKanyoro](https://github.com/RKanyoro)
+- Twitter: [@CarlosMbugua](https://twitter.com/CarlosMbugua)
+- LinkedIn: [Robert Mbugua](https://www.linkedin.com/in/robert-mbugua-a8745093/)
+
+👤 **Author1: Emmanuel Kipngeno**
+
+- GitHub: [@githubhandle](https://github.com/kkmanuu)
+- Twitter: [@twitterhandle](https://twitter.com/kkmanuu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emmanuel-kipngeno-879370242/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **backend**
+- [ ] **multi device support**
+- [ ] **sub tasks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give start and follow me on social media platforms listed above.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank @microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] Kindly fix this broken link, I got 404 error. Fixing this will make your repo to be more engaging. 👍